src/main/java/com/eerussianguy/firmalife/registry/ModRegistry.java

```diff
@@ -131,12 +166,37 @@ public static void registerBlocks(RegistryEvent.Register<Block> event)
         NormalIBs.add(register(r, "oven_wall", new BlockOvenWall(), CT_DECORATIONS));
         NormalIBs.add(register(r, "oven_chimney", new BlockOvenChimney(), CT_DECORATIONS));
 
+        FoodIBs.add(register(r, "pumpkin_fruit", new BlockStemFruit(), CT_FLORA));
+        FoodIBs.add(register(r, "melon_fruit", new BlockStemFruit(), CT_FLORA));
+
+        for(CropFL crop : CropFL.values())
+        {
+            String name = "crop/" + crop.name().toLowerCase();
```

Comment:
Same thing here. If it's the registry object for the dead crops that you need, I think you can get away with doing it elsewhere.